system/Debug/Timer.php

```diff
@@ -126,4 +126,22 @@ public function has(string $name): bool
     {
         return array_key_exists(strtolower($name), $this->timers);
     }
+
+    /**
+     * Executes callable and measures its time.
+     * Returns its return value if any.
+     *
+     * @param string   $name     The name of the timer
+     * @param callable $callable callable to be executed
+     *
+     * @return mixed|null
```

Comment:
Since this is a callable return it's fine to leave off the implied `null`. 
```suggestion
     * @return mixed
```

Reply:
See #6310
We should not add `mixed`.

Reply:
> See #6310 We should not add `mixed`.

The method returns the callable return value, which can be any type. Making this type specific might not be possible.

@kenjis What do you suggest?

Reply:
`object|array|string|int|float|bool|null`